docs/encyclopedia/application-message-passing.mdx

```diff
@@ -26,57 +28,85 @@ keywords:
+To define these, consider these messages from the perspective of the client making the request:
 
-## Signals {#signals}
+- Queries are read requests. They can read the current state of the Workflow but cannot block in doing so.
+- Signals are write requests. They cause changes in the running Workflow, but you cannot await any response or error.
+- Updates are interactive write requests, wherein the sender of the Update tracks completion and waits for a response or an error.
 
-**What is a Signal in the context of Temporal?**
+## How to choose between Signals, Updates, and Queries? {#choosing-messages}
 
-A Signal is an asynchronous request to a [Workflow Execution](/workflows#workflow-execution).
+### For write requests
 
-<RelatedReadContainer>
-    <RelatedReadItem path="/cli/workflow#signal" text="Send a Signal using the Temporal CLI" archetype="feature-guide" />
-    <RelatedReadItem path="/develop/go/message-passing#signals" text="Send and handle Signals with the Go SDK" archetype="feature-guide" />
-    <RelatedReadItem path="/develop/java/message-passing" text="Send and handle Signals with the Java SDK" archetype="feature-guide" />
-    <RelatedReadItem path="/develop/php/message-passing#signals" text="Send and handle Signals with the PHP SDK" archetype="feature-guide" />
-    <RelatedReadItem path="/develop/python/message-passing#signals" text="Send and handle Signals with the Python SDK" archetype="feature-guide" />
-    <RelatedReadItem path="/develop/typescript/message-passing#signals" text="Send and handle Signals with the TypeScript SDK" archetype="feature-guide" />
-    <RelatedReadItem path="/develop/dotnet/message-passing#define-a-signal" text="Send and handle Signals with the .NET SDK" archetype="feature-guide" />
-</RelatedReadContainer>
+From the above descriptions, it may sound like Updates can do all the things that Queries and Signals can do. But, unlike Signals, Updates must be synchronous, that is, they must wait for the worker running the Workflow to acknowledge the request.
+
+Use Signals instead of Updates when:
+
+- You want to send a fully asynchronous message.
+- Your client does not want to rely on the worker being available.
+- Your Workflow’s client cares more about the latency of sending the message than it does of the end-to-end operation.
+- When you need to [Signal-With-Start](signal-with-start) atomically start the Workflow and then Signal as the first operation. (since there is currently no Update-with-start operation.)
+
+Use Updates instead of Signals when:
+
+- You want to track the completion of the message from the client.
+- You need a result or an exception from your message without having to query subsequently.
+- You’d like to “validate” the Update before accepting it into the Workflow and its history.
+- Your client cares more about the latency of the end-to-end operation than you care about the latency of sending the message.
```

Comment:
As mentioned above, I'm not managing to understand these "cares mores about" bulletpoints. I've read several times and it's not becoming perfectly clear, so I think we need to drop them. (Also, who's "you"?)

```suggestion
```